File: python/TortillaHarinaTest.py
import unittest
from TortillaHarina import TortillaHarina

class TortillaHarinaTest(unittest.TestCase):
	tortillaHarina = TortillaHarina()

	def testCurrentTemperature(self):
		self.tortillaHarina.setCurrentTemperature(21)
		self.assertEqual(21, self.tortillaHarina.getCurrentTemperature())

	def testFalseToast(self):
		self.tortillaHarina.toast(False)
		self.assertFalse(self.tortillaHarina.isToasted())	

	def testTrueToast(self):
		self.tortillaHarina.toast(True)
		self.assertTrue(self.tortillaHarina.isToasted())

	def testToasting(self):
		self.assertEqual(30, self.tortillaHarina.getToastTemperature())

File: csharp/unittest-practice/src/test/TortillaHarinaTest.cs
using NUnit.Framework;
using unittestpractice.main;

namespace unittestpractice.test
{
    [TestFixture]
    internal class TortillaHarinaTest
    {
        private TortillaHarina _tortillaHarina;

        [SetUp]
        public void SetUp ()
        {
            _tortillaHarina = new TortillaHarina();
        }

        [TestCase]
        public void TestCurrentTemperature()
        {
            _tortillaHarina.SetCurrentTemperature(21);
            Assert.AreEqual(21, _tortillaHarina.GetCurrentTemperature());
        }

        [TestCase]
        public void TestFalseToas()
        {
            _tortillaHarina.Toast(false);
            Assert.IsFalse(_tortillaHarina.IsToasted());
        }

        [TestCase]
        public void TestTrueToast()
        {
            _tortillaHarina.Toast(true);
            Assert.IsTrue(_tortillaHarina.IsToasted());
        }

        [TestCase]
        public void TestToasting()
        {
            Assert.AreEqual(10, _tortillaHarina.GetToastTemperature());
        }
    }
}
